.gitignore

```diff
@@ -18,3 +18,6 @@ ci/install*
 ci/output*
 ci/flcl-x86*
 ci/flcl-ppc*
+
+# build directories in source tree
+/build*
```

Comment:
I am not administrating that repo but I typically do not like adding much to the `.gitignore` as this can be handled on the user side.
If you typically create directory prefixed with `build*` you can use `core.excludesFile` or `$GIT_DIR/info/exclude`

Reply:
ok fine. I'll remove that.